src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -0,0 +1,342 @@
+* The Kubernetes integration does not automatically update. For best results, regularly [update](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure#update).
+
+### Learn more [#update-kear-more]
+
+More resources for learning about configuration:
+
+* Learn [technical details about how configuration works](#config-details).
+* Learn how to [configure monitoring of multiple services with the same config file](#add-service).
+* See a [step-by-step tutorial showing how to monitor a Redis service on Kubernetes](/docs/monitor-service-running-kubernetes-tutorial).
+
+
+## Enable monitoring of services using manifests [#config-details]
+
+<Callout variant="tip">
+  We strongly encourage you to configure integrations through the `values-newrelic.yaml` file and our Helm Chart, as explained in the [section above](#enable). Configuring service monitoring on top of the manifest installation is substantially more difficult and offers no advantages whatsoever.
+</Callout>
+
+For each service you wish to monitor, you must add a configuration file for that integration to our Kubernetes integration's configuration. This document will cover these subjects:
+
+* [How the service-specific configuration YAML snippet works](#integration-config)
+* [Adding the service-specific YAML in the Kubernetes integration's config file](#configmap)
+* [Adding multiple services to the Kubernetes integration's config file](#add-service)
+
+### How the service-specific YAML config works [#integration-config]
+
+Our Kubernetes integration's configuration follows the [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) format. Using a `ConfigMap` allows us to decouple the configuration for the integrations from the Kubernetes image. The other benefit is that a `ConfigMap` can be updated automatically without reloading the running container.
+
+Because the infrastructure agent uses YAML to configure its associated integrations, `ConfigMaps` are a good choice for storing YAML. (For more information on config file format, see the [Integration config file format](/docs/integrations/integrations-sdk/file-specifications/integration-configuration-file-specifications).)
+
+The Kubernetes integration image comes with an [auto-discovery feature](https://github.com/newrelic/nri-discovery-kubernetes) that simplifies the configuration of multiple instances of services using a single configuration file. For example, if you have several NGINX instances running, creating an [NGINX integration configuration file](/docs/integrations/host-integrations/host-integrations-list/nginx-monitoring-integration#config) for every instance would be hard to implement and hard to update. With our auto-discovery option, you can discover and monitor all your NGINX instances with a single configuration file.
```

Comment:
NTH here as well a diagram could benefit the reader

Reply:
![image](https://user-images.githubusercontent.com/969721/151215064-c5883349-3958-49c5-8f8a-96d79ef61008.png)

😁